.cirrus.yml

```diff
@@ -167,8 +181,11 @@ task:
   info_script:
     - packer --version
 
-  build_image_script: |
+  export_date_script: |
     DATE=$(date --utc +'%Y-%m-%dt%H-%M-%S')
+    echo "DATE=${DATE}" | tee -a $CIRRUS_ENV
+
+  build_image_script: |
```

Comment:
I previously was thinking that it'd be nice to have the same timestamp for all images. But unfortunately it doesn't look like there's a proper CIRRUS_* variable for it. Perhaps we should suggest that as a feature?

Reply:
Yes, getting creation time of the task could be very helpful.

Reply:
Yes, please do.